### ISSUE TITLE
Bug changing connector size in navis.2d_plot()

### ISSUE THREAD

**Issue by mwinding** [navis-org/navis#93]
It seems the `cn_size` argument doesn't work as expected in navis.plot2d() anymore. See the below example where I change `cn_size=1` to `cn_size=100` with no effect. This is using navis 1.2.1 with Python 3.9.7 and macOS Big Sur 11.6.4. 

**Example**
nl = navis.example_neurons()
fig, ax = navis.plot2d(nl, method='3d_complex', connectors=True, **cn_size=1**, view=('x', '-y'))
plt.show()
<img width="339" alt="cn_size1" src="https://user-images.githubusercontent.com/2660187/165757102-3305b90c-e4a6-4aa2-b85c-43a8d3e78c9e.png">

nl = navis.example_neurons()
fig, ax = navis.plot2d(nl, method='3d_complex', connectors=True, **cn_size=100**, view=('x', '-y'))
plt.show()
<img width="340" alt="cn_size100" src="https://user-images.githubusercontent.com/2660187/165757125-74ad8bf7-04af-45d5-ac2d-46071e28580f.png">

**Comment by schlegelp** [comment]
Yeah not sure why/when that happened but it's already fixed on Github (6441b7c7eae6cd6c5758aaca7e0425a57cc48ad3). I haven't released a new version since that fix so you'd need to re-install from Github for now.

**Comment by mwinding** [comment]
Oh perfect. I'll give that a go then, thanks!
